### ISSUE TITLE
Page contents the plugin syntax can not be loaded

### ISSUE THREAD

**Issue by Autumnf0x** [saggi-dw/dokuwiki-plugin-stepbystep#1]
### Page contents the plugin syntax can not be loaded:
  * Just installed the plugin, using the Extension Manager.
  * The page containing the content of the plugin does not load completely and freezes, while the CPU load increases greatly (the fan howls).
 
> Dokuwiki Release 2023-04-04a "Jack Jackrum";
> PHP 8.2;
> StepByStep Plugin last updated on 2023-07-27;
> `defer_js` set to true;

**Comment by saggi-dw** [comment]
Hi, thank you for pointing this out. 
Can you please provide a list of installed plugins and the source of your page content?

**Comment by Autumnf0x** [comment]
> Hi, thank you for pointing this out.
> Can you please provide a list of installed plugins and the source of your page content?
---
Sure, here is a list of plugins I have installed (bundled plugins are not include):

  - Addnewpage
  - Backup
  - Batchedit
  - Bureaucracy
  - Cloud
  - Custombuttons
  - Dropfiles
  - Dw2pdf
  - Edittable
  - Gallery
  - Hidden
  - Imagebox
  - Include
  - Move
  - Newpagetemplate
  - Nspages
  - Odt
  - Pagelist
  - Smtp
  - Tag
  - Tocselect
  - Todo
  - Wrap
  - Yalist

Here is a source of page with stepbystep plugin syntax:
 ```
 ====== PlayGround ======

 #: Title of the hidden content
 Hidden content goes here
 :#

 ```

**Comment by saggi-dw** [comment]
Thank you. There seems to be an overlap with the BackupTool for DokuWiki. I will adjust the class names(`collapsible`).

**Comment by saggi-dw** [comment]
Should be fixed with the 06.08.23 version. Since the class names were changed the cache must be deleted so that the styles become active.

**Comment by Autumnf0x** [comment]
> the cache must be deleted

How to do it?

**Comment by saggi-dw** [comment]
https://www.dokuwiki.org/caching#purging_the_cache

simply resaving the configuration from the [Configuration Manager](https://www.dokuwiki.org/plugin:config).

**Comment by Autumnf0x** [comment]
> Should be fixed with the 06.08.23 version. Since the class names were changed the cache must be deleted so that the styles become active.

> https://www.dokuwiki.org/caching#purging_the_cache
> 
> simply resaving the configuration from the [Configuration Manager](https://www.dokuwiki.org/plugin:config).

Yes, it already works! Thanks!

**Comment by saggi-dw** [comment]
Thank you for the Feedback.
